tests/PintaBenchmarks/AdjustmentsBenchmarks.cs

```diff
@@ -7,90 +7,90 @@ namespace PintaBenchmarks;
 
 [MemoryDiagnoser]
 [Config (typeof (MillisecondConfig))]
-public class AdjustmentsBenchmarks
+public sealed class AdjustmentsBenchmarks
 {
 	private readonly ImageSurface surface;
 	private readonly ImageSurface dest_surface;
-	private readonly RectangleI bounds;
```

Comment:
Was there a reason for changing this variable name? `bounds` seemed reasonable to me

Reply:
It was just to make it evident that it's not a single rectangle, but that it was wrapped in an array. But no problem, I can change it back

Reply:
I just changed it back 👍